torch/_refs/__init__.py

```diff
@@ -1319,6 +1320,15 @@ def _trunc_divide(
 #
 
 
+@out_wrapper
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("self", "tensor1", "tensor2"),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+)
+def addcdiv(self: Tensor, tensor1: Tensor, tensor2: Tensor, value: float = 1):
```

Comment:
If value is a float, why are you assigning an integer to it? Also, isn't value a NumberType?

Reply:
this is now addressed as in the other refs: doing it any other way seems to make the linter unhappy, which is why it's None in the type signature and not some number